src/panel-triggers/triggers_panel_ctrl.js

```diff
@@ -62,6 +62,7 @@ export const PANEL_DEFAULTS = {
   statusIcon: false,
   severityField: true,
   ageField: false,
+  dataSourceField: true,
```

Comment:
I don't think it makes sense to enable it by default, it helpfull for multi-datasource panel which is not enabled by default.

Reply:
I agree, I've made it false at the beginning, but then joining that option together with list layout made me change that to true...